src/CacheEntryPool/CacheEntryPool.php

```diff
@@ -56,7 +56,6 @@ class CacheEntryPool implements CacheItemPoolInterface
     protected const HASH_DB_PREFIX = 'Cache_Pool';
 
     /**
-     * @todo use Psr\SimpleCache\CacheInterface ?
```

Comment:
it is derived from the simple `CacheInterface` of Psr group, but isn't simple anymore